src/Tizen.Multimedia.Util/Interop/Interop.ThumbnailExtractor.cs

```diff
@@ -46,6 +46,13 @@ internal static class ThumbnailExtractor
 
         [DllImport(Libraries.ThumbnailExtractor, EntryPoint = "thumbnail_util_destroy")]
         internal static extern ThumbnailExtractorError Destroy(IntPtr handle);
+
+        [DllImport(Libraries.ThumbnailExtractor, EntryPoint = "thumbnail_util_extract_to_buffer")]
+        internal static extern ThumbnailExtractorError ExtractToBuffer(string path, uint width, uint height, IntPtr thumbData,
```

Comment:
Why not `out IntPtr thumbData` ?

Reply:
It's the same case with https://github.com/Samsung/TizenFX/pull/919.

Reply:
Right, it's not same case with https://github.com/Samsung/TizenFX/pull/919.
I'll change it.

Reply:
It seem to not same. it is just outparam

Reply:
I changed and pushed it already.